gammapy/makers/tests/test_reduce.py

```diff
@@ -154,9 +154,17 @@ def makers_spectrum(exclusion_mask):
             "n_jobs": 2,
             "backend": "multiprocessing",
         },
+        {
+            "dataset": get_mapdataset(name="parallel_staking"),
+            "stack_datasets": True,
+            "cutout_width": None,
+            "n_jobs": 2,
+            "backend": "ray",
+        },
     ],
 )
 @requires_data()
+@requires_dependency("ray")
```

Comment:
Is it possible to have requires_dependency("ray") only for one of the parametrized tests ? or it has to be written separately ?

Reply:
Probably it's possible. One can use `pytest.skip()` and an in-line if statement. But I think it is easier to just split out the test in this case. The benefit is that the test will not be collected, the test printout becomes more meaningful when the test is not skipped at execution.